bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/LutronBindingConstants.java

```diff
@@ -73,6 +73,11 @@
     public static final String SERIAL_NUMBER = "serialNumber";
     public static final String DISCOVERY_FILE = "discoveryFile";
 
+    public static final String PROPERTY_PRODFAM = "productFamily";
+    public static final String PROPERTY_PRODTYP = "productType";
+    public static final String PROPERTY_CODEVER = "version";
+    public static final String PROPERTY_MACADDR = "macAddress";
```

Comment:
There are global constants available for some properties like this (see [Thing.java](https://github.com/openhab/openhab-core/blob/60e040c5296c20a0acd2d4b7e54c202f186edf7f/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L58)).

Reply:
Ok. The ones that are just properties I changed to use the global constants from Thing. The ones that are user-settable parameters I left using local definitions.